tests/e2e/tests_runner.sh

```diff
@@ -107,6 +107,11 @@ main() {
 
 	cd "${tests_repo_dir}/integration/kubernetes/confidential"
 
+        # Update runtimeclass and kata default config for the scripts in tests repo
+        sed -i "s/runtimeClassName: kata/runtimeClassName: $runtimeclass/g" fixtures/*.yaml
+	local default_config_link="/opt/confidential-containers/share/defaults/kata-containers/configuration.toml"
+        sed -i "s#kata-runtime kata-env#kata-runtime --config $default_config_link kata-env#g" ../../../lib/common.bash
```

Comment:
I didn't understand how that fix the other problem. Tests will need to change the configuration.toml that matches the runtimeclass, e.g., kata-qemu -> configuration.toml, kata-tdx-qemu -> configuration-tdx-qemu.toml...etc...

Reply:
The tests scripts will try to modify the kata root image and use `kata-runtime kata-env` to get the image path.
`kata-runtime` will search the default config path, but our CoCo installed config path is `/opt/confidential` which is not in the default config path lists.
```
kata-runtime --show-default-config-paths
/etc/kata-containers/configuration.toml
/usr/share/defaults/kata-containers/configuration.toml
```
Then we need pass our CoCo runtime config to the scripts, else we need change the config file in kata default path.

Reply:
In my VM with CoCo installed via operator, it gets the path correctly:
```
vagrant@ubuntu2004:~$ export PATH=$PATH:/opt/confidential-containers/bin
vagrant@ubuntu2004:~$ kata-runtime --show-default-config-paths
/etc/kata-containers/configuration.toml
/opt/confidential-containers/share/defaults/kata-containers/configuration.toml
```
Maybe in your environment you get `/usr/share/...` because you had kata from CoCo and non-CoCo (ie. from Kata CI) installed together and the kata-runtime (a symlink?) points to the wrong installation?